src/main/java/com/toptrumps/core/player/AIPlayer.java

```diff
@@ -23,9 +26,11 @@ public AIPlayer(PlayerState playerState) {
 
     public Attribute selectAttribute() {
         Card topCard = getTopCard();
-        Attribute highestAttribute = topCard.getHighestAttribute();
-        this.setSelectedAttribute(highestAttribute);
-        return highestAttribute;
+        ArrayList<Attribute> highestAttributes = topCard.getHighestAttributes();
+        Random rand = new Random();
```

Comment:
We've been generating random numbers for a while now. I suggest we extract this random number generator out of this method.